lib/graphql/lang/visitor_context.ex

```diff
@@ -0,0 +1,6 @@
+
+defmodule GraphQL.Lang.VisitorContext do
```

Comment:
Visitors can be provided with extra information during traversal. One such piece of information is the stack of nodes of the AST. In the future it will contain a `TypeInfo` map that would have been generated from an earlier pass.  Work in progress.